README.md

```diff
@@ -35,6 +37,21 @@ differences:
   authenticating to the registry. Must be specified for private repos or when
   using `put`.
 
+* `ecr`: *Optional. Default `false`.* If set, Access Key ID and secret will be 
+  used to authenticate, `username` and `password` will be ignored.
+
+* `aws_access_key_id`: *Optional. Default `"""`.* If set, will be used to set
+    `AWS_ACCESS_KEY_ID` environment variable
```

Comment:
Setting an env var is an implementation detail and doesn't really explain the intent of the param - could this just say something like 'access key ID to use for accessing ECR'? (same for the rest)